### ISSUE TITLE
Trailing space after escaped pipe character is being removed when user saves the feature file

### ISSUE THREAD

**Issue by himanshuG-92** [alexkrechik/VSCucumberAutoComplete#444]
**Describe the bug**
I have some scenario where I need to match string which contain pipes with whitespaces both the side, but when I am trying to escape pipe then trailing white space which I need to have it in the string is also removed.

**Expected behavior**
```
Scenario Outline : Verify passed string
        Given Some situation
        When I do some stuff with "<title>"
        Then expected thing happens
Examples:
       |title         |
       |abc \| xyz|

```
**Actual behavior**
```
Scenario Outline : Verify passed string
        Given Some situation
        When I do some stuff with "<title>"
        Then expected thing happens
Examples:
       |title        |
       |abc \|xyz|

```

**Issue**
While saving the feature file, trailing whitespace after escape character (pipe '|' ) is removed but as it should not be. As scenario needs to have string  "abc | xyz" as input.

**Comment by alexkrechik** [comment]
Should be fixed after the 2.16.0 release

**Comment by meskmanz** [comment]
Does some workaround exist?

**Comment by VictorVermeulen** [comment]
> Does some workaround exist?

Did you find something?

**Comment by evozniak** [comment]
> Does some workaround exist?

For me only saving without formatting.

**Comment by viktor-silakov** [comment]
I have same issue

**Comment by andersonvcv** [comment]
When will it be delivered?

**Comment by andersonvcv** [comment]
Bump @alexkrechik

**Comment by alexkrechik** [comment]
> Bump @alexkrechik

Unfortunately, numerous issues had piled up due to outdated libraries, making local testing of new features impossible. Consequently, I had to address https://github.com/alexkrechik/VSCucumberAutoComplete/issues/483. I should retest all the scenarios and make some additional refactoring. But the libs update is done, and tests are up.

**Comment by alexkrechik** [comment]
This issue is to be fixed or implemented in the 3.0.0 version of the extension. If this issue is unresolved or there are additional notes, please create a new ticket with the necessary details.
